src/test/java/UploadReceiptServletTest.java

```diff
@@ -394,6 +381,42 @@ public void doPostThrowsIfReceiptAnalysisFails() throws IOException {
     verify(blobstoreService).delete(BLOB_KEY);
   }
 
+  @Test
+  public void doPostRoundPrice() throws IOException {
+    helper.setEnvIsLoggedIn(true);
+
+    StringWriter stringWriter = new StringWriter();
+    PrintWriter writer = new PrintWriter(stringWriter);
+    when(response.getWriter()).thenReturn(writer);
```

Comment:
Why is this needed for this test but not for doPostUploadsReceiptToDatastoreLiveServer?